bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -762,13 +763,13 @@ impl ParseError {
 
         match self {
             InvalidInteger { source, input } if *source.kind() == IntErrorKind::PosOverflow => {
-                write!(f, "{} {} is above limit {}", subject, input, upper_bound)
+                write!(f, "{} {} is above limit {}", subject, input.display_raw(), upper_bound)
```

Comment:
Oh, now I see why this and why there are two PRs. This should display `{subject} is above limit {}` without alloc and `{subject} "{input.as_str()}" is above limit` with alloc. Same below.